flaps/flaps.go

```diff
@@ -212,13 +216,14 @@ func snakeCase(s string) string {
 	})
 }
 
-func (f *Client) _sendRequest(ctx context.Context, action flapsAction, method, endpoint string, in, out interface{}, headers map[string][]string) error {
-	actionName := snakeCase(action.String())
+func (f *Client) _sendRequest(ctx context.Context, method, endpoint string, in, out interface{}, headers map[string][]string) error {
+	actionName := snakeCase(actionFromContext(ctx).String())
 
 	ctx, span := tracing.GetTracer().Start(ctx, fmt.Sprintf("flaps.%s", actionName), trace.WithAttributes(
 		attribute.String("request.action", actionName),
 		attribute.String("request.endpoint", endpoint),
 		attribute.String("request.method", method),
+		attribute.String("request.machine_id", machineIDFromContext(ctx)),
```

Comment:
Is this how we're recording "machine id" on the flyd and flaps side? Ignore me if it doesn't make sense(I'm bad at tracing). Mostly wondering if we can track this across services.